### DISCUSSION TITLE
Keda unable to scale up pods when idleReplicaCount is 0 on kafka

### DISCUSSION THREAD

**Discussion by Nwayes** [kedacore/keda#6220] [Q&A / Need Help]
Hello,
Here is my context.
We are using Keda to autoscale pods on AKS.
The pods will consume from a Confluent kafka topic.
This is my actual config :

  idleReplicaCount: 0
  maxReplicaCount: 5
  minReplicaCount: 1
  pollingInterval: 10
  activationLagThreshold: '1'
  lagThreshold: '50'
  
With this configuration, if there is no messages in the topic, the pods will be scaled down to 0, which is expected.
If we have messages in the topic, the pods will not be scaled up.
If i put minReplicaCount: 0 and i have only 20 or 30 messages, the pods will not scale up (which is expected)
This is why i need the  idleReplicaCount: 0
So any suggestions on what I may be missing?

**Comment by zroubalik** [comment]
make sure that all partitions have commited offset
